client/components/LandingHeader.tsx

```diff
@@ -1,13 +1,19 @@
 import styles from "../styles/landingHeader.module.css";
 import Button from "../components/Button";
+
+interface LandingHeaderProps {
+    onClickHost: any;
+    onClickJoin: any;
```

Comment:
Avoid explicit `any`, should be function type.

Reply:
Not sure how to do this and the docs are a little unclear?

Reply:
In this case the type signature is `() => void` because `Button#onClick` takes no arguments and returns no value. A function that adds two numbers ands returns them would be `(x: number, y: number) => number`.

```ts
interface LandingHeaderProps {
    onClickHost: () => void;
    onClickJoin: () => void;
}
```

Reply:
I see you define the function's types instead of the function. Neat!